_explore/scripts/build_yearlist.py

```diff
@@ -1,11 +1,12 @@
 from scraper.github import queryManager as qm
+from os import environ as env
 import os.path
 
 yearDict = {}
 
 # Gather all file name data
 print("Checking GitHub data file names with year stamps...")
-for file in os.listdir("../github-data"):
+for file in os.listdir(env['GITHUB_DATA']):
```

Comment:
You might want to consider doing something like `env.get('GITHUB_DATA', '../github-data')` or similar to set a default base directory in the event the environment variable is missing.

Reply:
Got it, adding via #120